src/edu-benefits/schema.js

```diff
@@ -280,7 +280,7 @@ module.exports = {
           }
         }
       },
-      required: ['commissionYear', 'rotcScholarshipAmounts']
+      required: ['rotcScholarshipAmounts']
```

Comment:
This won't force any required fields, it just means the scholarship list has to be there. By default, the front end will send back an empty list.